internal/datastore/common/schema.go

```diff
@@ -77,6 +85,36 @@ func (si SchemaInformation) debugValidate() {
 	}, "SchemaInformation failed to validate")
 }
 
+func (si SchemaInformation) sortByResourceColumnOrderColumns() []string {
```

Comment:
add unit test to assert custom/default values

Reply:
Added